src/ResourceManager/Resources/Commands.Resources/Models.Authorization/AuthorizationClient.cs

```diff
@@ -525,7 +518,7 @@ private static void ValidateRoleDefinition(PSRoleDefinition roleDefinition)
                 throw new ArgumentException(ProjectResources.InvalidAssignableScopes);
             }
 
-            if (roleDefinition.Actions == null || !roleDefinition.Actions.Any())
+            if ((roleDefinition.Actions == null || !roleDefinition.Actions.Any()) && (roleDefinition.DataActions == null || !roleDefinition.DataActions.Any()))
```

Comment:
Will this cause any scripts to fail?

Reply:
No.
earlier the condition was if they had no value actions,it used to error out.
but now the condition is that if they dont have any value for actions and also no value for datactions,it will error out